subsys/bluetooth/gatt_pool.c

```diff
@@ -73,6 +73,10 @@ static struct svc_el_pool chrc_pool = {
 	.locks = BT_GATT_CHRC_LOCKS,
 };
 
+static const struct bt_uuid *uuid_primary = BT_UUID_GATT_PRIMARY;
```

Comment:
``static const struct * const uuid_primary`` to be on complete safe state.

The name here suggest that this is something from uuid module. Personally I would preffer something like gatt_pool_uuid_chrc. But this is not a strong opinion.

Reply:
Good point, thank you.